test/index.js

```diff
@@ -96,6 +97,48 @@ describe('KISSmetrics', function () {
     });
   });
 
+  describe.only('.completedOrder()', function(){
+    it('should send a completed order call correctly', function(done){
+      settings.prefixProperties = true;
+      var json = test.fixture('track-ecommerce');
+
+      test
+        .requests(3)
+        .request(0)
```

Comment:
@yields Couldn't figure out how the multi request tests work, we need to verify that three requests were made, and verify the right query parameters were set on each of them. How would we do that?